Nishant/Day_12/postgres assignment/src/controller/general.ts

```diff
@@ -0,0 +1,59 @@
+import { Context } from "vm";
+
+import * as servicegeneral from '../services/general';
+
+interface topper{
+    student_id:number;
+    fname:string;
+    marks:number;
+}
+
+export async function gettopperByclassIdAndSubjectId(ctx: Context){
+    try{
+        var c_id=parseInt(ctx.params.c_id);
```

Comment:
use let and use object de structuring here
link for de structuring https://javascript.info/destructuring-assignment